docs/design/datacontracts/RuntimeTypeSystem.md

```diff
@@ -137,6 +137,31 @@ partial interface IRuntimeTypeSystem : IContract
     // Return true if a MethodDesc represents an IL Stub dynamically generated by the runtime
     // A IL Stub method is also a StoredSigMethodDesc, and a NoMetadataMethod
     public virtual bool IsILStub(MethodDescHandle methodDesc);
+
+    // Return true if a MethodDesc is in a collectible module
+    public virtual bool IsCollectibleMethod(MethodDescHandle methodDesc);
+
+    // Return true if a MethodDesc is in a module that supports Edit-And-Continue
+    public virtual bool InEnCEnabledModule(MethodDescHandle methodDesc);
```

Comment:
I don't like this. We have a scheme to go from MethodDesc to Module... and we already have a contract there that exposes whether or not the Module is Edit and Continue aware.

Reply:
hmm... this looks like dead code actually.  `CodeVersionManagerSupportsMethod` already does the MethodDesc->MethodTable->Module->ModuleFlags dance.